ui_framework/components/event/_event.scss

```diff
@@ -13,14 +13,14 @@
   flex: 1 1 0%;
 }
 
-  .kuiEventBody__message {
-    font-size: $globalFontSize;
-    line-height: $globalLineHeight;
-    color: $globalFontColor;
-  }
+.kuiEventBody__message {
```

Comment:
Can we keep these indented? See https://github.com/elastic/kibana/blob/master/style_guides/css_style_guide.md#create-descendant-classes-to-represent-child-components-which-cant-stand-on-their-own for more context.